core/server/data/meta/schema.js

```diff
@@ -69,6 +69,8 @@ function getPostSchema(metaData, data) {
     var description = metaData.excerpt ? escapeExpression(metaData.excerpt) : null,
         schema;
 
+    const context = data.page ? 'page' : 'post';
```

Comment:
Was thinking of adding a parameter `context` which would be for the caller - `getSchema` to specify, but didn't go that route as it's not really guaranteed what the properties would be present if e.g. [amp](https://github.com/TryGhost/Ghost/pull/10718/files#diff-50462e912decc8ded2e96b6f5e46e31cR178) condition kicks in. Additionally, the logic below depends on the post/page property being present and the 'context' check - `_.includes(context, 'post')` is not directly guaranteeing that I think. 

These ifs for post or page are another sign of issues with meta layer :man_shrugging: